tests/main/mount-order-regression/task.yaml

```diff
@@ -0,0 +1,57 @@
+summary: Regression test for snap-update-ns mount order
+
+environment:
```

Comment:
The environment should go after the details

Reply:
Oops, thanks!